### PR TITLE
Bookings Showing Page

### PR DESCRIPTION
**Pull request by Ashik7860**
Booking Card

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
   return (
-    <Routes>
-      <Route path="/*" element={<ProviderRoutes/>} />
-    </Routes>
+    <div>
+      <Routes>
+        <Route path="/*" element={<ProviderRoutes />} />
+      </Routes>
+
+      <div className="p-4">
```
Comment (JaseelaAlimahamed) [inline src/App.jsx]: no need to give app.js

File: src/components/reUsableComponents/AppointmentCard.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { CalendarIcon, ClockIcon } from '@heroicons/react/24/outline'; // Import Heroicons
```
Comment (JaseelaAlimahamed) [inline src/components/reUsableComponents/AppointmentCard.jsx]: you should avoid extra packages

File: src/App.jsx (hunk 1 above)
Comment (Ashik7860) [reply src/App.jsx]: Updated

File: src/components/reUsableComponents/AppointmentCard.jsx (hunk 2 above)
Comment (Ashik7860) [reply src/components/reUsableComponents/AppointmentCard.jsx]: Avoided extra packages

File: package.json [hunk 3]
```diff
@@ -10,6 +10,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@heroicons/react": "^2.1.5",
```
Comment (JaseelaAlimahamed) [inline package.json]: y you add this package?

File: src/routes/ProviderRoutes.jsx [hunk 4]
```diff
@@ -9,7 +10,9 @@ function ProviderRoutes() {
       <Routes>
         
         <Route path="/" element={<Navigate to="/signin" />} />
-
+        
+        <Route path="/Booking" element={ <CardComponent/>} />
+        <Route path="*" element={<Navigate to="/Booking" />} />
```
Comment (JaseelaAlimahamed) [inline src/routes/ProviderRoutes.jsx]: no need of this route

**Review by JaseelaAlimahamed — APPROVED** [review]

**Pull request merged**